consensus/wal_test.go

```diff
@@ -93,7 +93,7 @@ func benchmarkWalDecode(b *testing.B, n int) {
 	enc := NewWALEncoder(buf)
 
 	data := nBytes(n)
-	enc.Encode(&TimedWALMessage{Msg: data, Time: time.Now().Round(time.Second)})
+	enc.Encode(&TimedWALMessage{Msg: data, Time: time.Now().Round(time.Second).UTC()})
```

Comment:
`tmtypes.Now()`

Reply:
Missed it.

Reply:
The difference seems to be `.Round(time.Second)` where in `types.Now()` we do `Round(0)`

Reply:
Is there some reason we need to round to the nearest second? Or let's just use tmtypes.Now ?